apps/frontend/src/app/components/header/header.component.scss

```diff
@@ -205,10 +205,14 @@
         margin: 0;
       }
     }
+    a {
+      display: inline-block;
+    }
   }
 
   &.open {
-    &:hover .submenu,
+    &:hover,
```

Comment:
This should remain `&:hover .submenu` because it was one rule, that kept open the submenu in mobile view